wrdcld/cli.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+
+
+def parse_args():
```

Comment:
just for my own understanding more than anything,  what are the benefits of having a CLI rather than just using scripts? Is it a matter of preference or?